app/v2/connectors/PersistenceConnector.scala

```diff
@@ -212,6 +236,7 @@ class PersistenceConnectorImpl @Inject() (httpClientV2: HttpClientV2, appConfig:
     httpClientV2
       .patch(url"$url")
       .withBody(Json.toJson(body))
+      .setHeader(Constants.XMessageTypeHeader -> messageType.code)
```

Comment:
The patch call doesn't need this as the message type will be in the body (but it doesn't harm keeping it).